tests/commandline/test.py

```diff
@@ -177,7 +177,6 @@ def runTest():
     ["test_plusname.csd", "test +Name for instr name"],
     ["testnewline.csd", "test newline in statements"],
     ["testmidichannels.csd", "test use of mapped multiport channels"],
-    ["test_max_table_len.csd", "test max table length"],
```

Comment:
Looks like this test was removed from the suite, was this intentional?

Reply:
yes, I moved it to a unit test as I could make it test the difference between float build and double build better.

Reply:
Probably should delete the unused test csd then

Reply:
Maybe the current test_max_table_len.csd can be moved to the manual as an example to use for floatsize?